ci/check_style.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 # Copyright (c) 2020-2024, NVIDIA CORPORATION.
 
+exit 0
+
```

Comment:
```suggestion
```

Reply:
🙈 ugh this was left behind from debugging and I missed it, thank you.